src/Microsoft.ML.Data/Utilities/SlotDropper.cs

```diff
@@ -211,7 +210,7 @@ public void DropSlots<TDst>(ref VBuffer<TDst> src, ref VBuffer<TDst> dst)
                 Contracts.Assert(index <= max);
             }
 
-            dst = new VBuffer<TDst>(newLength, iiDst, values, indices);
+            dst = mutation.CreateBuffer(iiDst);
```

Comment:
>iiDst [](start = 40, length = 5)

this looks like the only usage of this parameter.
Can we maybe have a separate version of `CreateBuffer / Commit`? Like, I don't know, `CommitTruncated` or something?

Reply:
This isn't the only usage of the parameter. I pointed out one above in the other comment. And there are more, like in EnsembleUtils.cs.